### PR TITLE
Video player fixes, added missing binders

### PR DESCRIPTION
**Pull request by adrianboguszewski**

### CODE REVIEW

**Comment by paularamo** [comment]
Adrian, Could you please add a screenshot of the output of the model with an example of one question one answer, in the readme files? Also in the notebook readme file.

**Comment by adrianboguszewski** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -47,13 +47,13 @@ Demos that demonstrate inference on a particular model.
...
-| [208-optical-character-recognition](notebooks/208-optical-character-recognition/)<br> | Annotate text on images using text recognition resnet | <img src="https://user-images.githubusercontent.com/36741649/129315292-a37266dc-dfb2-4749-bca5-2ac9c1e93d64.jpg" width=225> |
+| [207-vision-paddlegan-superresolution](notebooks/207-vision-paddlegan-superresolution/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F207-vision-paddlegan-superresolution%2F207-vision-paddlegan-superresolution.ipynb) | Upscale small images with superresolution using a PaddleGAN model| |
+| [208-optical-character-recognition](notebooks/208-optical-character-recognition/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F208-optical-character-recognition%2F208-optical-character-recognition.ipynb) | Annotate text on images using text recognition resnet | <img src="https://user-images.githubusercontent.com/36741649/129315292-a37266dc-dfb2-4749-bca5-2ac9c1e93d64.jpg" width=225> |
```
Comment (helena-intel) [inline README.md:51]: Have you tested 208 on Binder? I tried a few times and it always failed. 
image

File: notebooks/README.md [hunk 2]
```diff
@@ -37,13 +37,13 @@ Demos that demonstrate inference on a particular model.
...
-| [208-optical-character-recognition](208-optical-character-recognition/)<br> | Annotate text on images using text recognition resnet | <img src="https://user-images.githubusercontent.com/36741649/129315292-a37266dc-dfb2-4749-bca5-2ac9c1e93d64.jpg" width=225> |
+| [207-vision-paddlegan-superresolution](207-vision-paddlegan-superresolution/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F207-vision-paddlegan-superresolution%2F207-vision-paddlegan-superresolution.ipynb) | Upscale small images with superresolution using a PaddleGAN model| |
+| [208-optical-character-recognition](208-optical-character-recognition/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F208-optical-character-recognition%2F208-optical-character-recognition.ipynb) | Annotate text on images using text recognition resnet | <img src="https://user-images.githubusercontent.com/36741649/129315292-a37266dc-dfb2-4749-bca5-2ac9c1e93d64.jpg" width=225> |
```
Comment (helena-intel) [inline notebooks/README.md:41]: See comment on main README about 208 on Binder.

File: README.md (hunk 1 above)
Comment (adrianboguszewski) [reply README.md:51]: Yes, it worked for me

Comment (adrianboguszewski) [reply README.md:51]: Ok, it looks like it worked on Friday, but it doesn't work now. There is too little memory.

**Review by raymondlo84 — APPROVED** [review]
Yes, the patch looks good! It solved the virtual webcam FPS = 0 issue.

**Comment by raymondlo84** [comment]
docker_nbval / build (pull_request) Failing after 53m — build

Oh dear, why is that failing? @adrianboguszewski ?

**Comment by adrianboguszewski** [comment]
No idea. It's not related to my changes

**Comment by adrianboguszewski** [comment]
It's already merged. Let's add it in the next PR.

**Pull request merged**